metaflow/tagging_util.py

```diff
@@ -29,6 +30,12 @@ def is_utf8_decodable(x):
 # How long may an individual tag value be
 MAX_TAG_SIZE = 500
 
+def validate_aws_tag(tag):
+    validate_tags([tag['key'], tag['value']])
```

Comment:
`validate_tags` is for metaflow specific metadata tags validation. Some of the rules, like max tag set size probably do not apply to aws tags. Does it make sense to reuse the metaflow tag validation here, or can we do without?

It makes more sense to me that the AWS tag validation be kept completely separate

Reply:
was this the reason that the tags were turned into a more complex type?